src/codewhisperer/service/securityIssueHoverProvider.ts

```diff
@@ -94,10 +126,24 @@ export class SecurityIssueHoverProvider implements vscode.HoverProvider {
      * @returns The markdown string
      */
     private _makeCodeBlock(code: string, language?: string) {
-        // Add some padding so that each line has the same number of chars
         const lines = code.split('\n').slice(1) // Ignore the first line for diff header
+        // Get the length of the longest line to pad each line to be the same length
         const maxLineChars = lines.reduce((acc, curr) => Math.max(acc, curr.length), 0)
-        const paddedLines = lines.map(line => line.padEnd(maxLineChars + 2))
+        // Get the number of leading whitespaces that can be removed to hide unnecessary indentation
+        const minLeadingWhitespaces = lines.reduce((acc, curr) => {
```

Comment:
would you mind generalizing this as a `dedent()` function which can live next to [indent()](https://github.com/aws/aws-toolkit-vscode/blob/4a9fea7bd2383a9f9bd85839d57acc98a98fdba1/src/shared/utilities/textUtilities.ts#L40)

Reply:
After more consideration I decided to remove this, as it might cause some confusion about indentation in the suggested fix (especially in python where indentation is important)